submit.go

```diff
@@ -165,14 +165,26 @@ type submitResponse struct {
 	ReportURL string `json:"report_url,omitempty"`
 }
 
+type submitErrorResponse struct {
+	Error string `json:"error"`
+	ErrorCode string `json:"errcode"`
+}
+
+type submitPolicyErrorResponse struct {
+	Error string `json:"error"`
+	ErrorCode string `json:"errcode"`
+	PolicyUrl string `json:"policy_url,omitempty"`
+}
```

Comment:
do we need both of these? can't we just share one? Which would also mean we can use `writeError` for the policy errors too?

Reply:
Have done so. Noting that this is go 1.16 which I believe is before generics are a thing, so I have used what I think is "any struct" which does the trick.

Reply:
We still have definitions of both `submitPolicyErrorResponse` and `submitErrorResponse`. For clarity, I am suggesting that we just do:

```suggestion
type submitErrorResponse struct {
	Error string `json:"error"`
	ErrorCode string `json:"errcode"`
	PolicyURL string `json:"policy_url,omitempty"`
}
```

... and we set PolicyURL to "" where it is unused.